input/blog/2024-11-12-cake-v5.0.0-released.md

```diff
@@ -0,0 +1,89 @@
+﻿---
+title: Cake v5.0.0 released
+category: Release Notes
+author: devlead
+releaseName: 5.0.0
+---
+
+Version **5.0.0** of Cake has been released. Take it for a spin and give us feedback on our [discussion board](https://github.com/orgs/cake-build/discussions/4397).
+
+This release includes new features, improvements and bug fixes to both [Cake Scripting](/docs/running-builds/runners/dotnet-tool) and [Cake Frosting](/docs/running-builds/runners/cake-frosting) since the [Cake v4.2.0 release](/blog/xxxx/xx/cake-v4.2.0-released)! 🚀 🍰
+
+### Highlights of this release
+
+- .NET 9 support & C# 13 support
+- FreeBSD support
+- Addin recommended Cake version set to 5.0.0
+- 100 million downloads of Cake.Tool
+
+
+### .NET 9 support & C# 13 support
```

Comment:
Should explicitly mention that we no longer support .NET 6 and .NET 7?

Something like we did in the Cake 3.0 blog post: https://cakebuild.net/blog/2022/11/cake-v3.0.0-released

Reply:
Added this blurb

> Cake itself will no longer run on .NET 6 or .NET 7, but Cake will continue to support the building of .NET Framework projects, as well as projects targeting .NET 7.0 or older.